tests/storage-disks-vm

```diff
@@ -221,8 +175,67 @@ else
   echo 'Skipping NVME test due to missing extension: "disk_io_bus"'
 fi
 
+
+# Check adding a disk to a vm with a long name (max 63) and slash to test possible problems with long socket paths or long qemu device tags
+LONG_DEVICE_NAME="device-/-with-very-long-name-and-4-qemu-property-handling-test_"
+DEVICE_HOTPLUG="1"
+
+lxc launch "${IMAGE}" v2 --vm
+
+echo "==> Check /dev/disk/by-id with escaped long device name"
+waitInstanceReady v2
+lxc config device add v2 ${LONG_DEVICE_NAME} disk source="${loopdev}"
+[ "$(lxc exec v2 -- stat -c "%F" /dev/sdb)" = "block special file" ]
```

Comment:
This is still hardcoding `/dev/sdX` which is what we should avoid. I'd re-order the test to have a `readlink` call to know which `/dev/sdX` it points to and then use that variable later on.

Reply:
I was just making this exact change :). Will be pushing it shortly, thanks!

Reply:
I was just making this exact change :). Will be pushing it shortly, thanks!